templates/layouts/_submit.html

```diff
@@ -23,7 +23,7 @@ <h1 class="ons-u-mt-m">{{ content.title }}</h1>
 
   {% if content.guidance %}
     <div class="ons-u-mt-l">
-      <p class="ons-u-mb-no" data-qa="guidance">{{ content.guidance }}</p>
+      <p class="ons-u-fs-r--b" data-qa="guidance">{{ content.guidance }}</p>
```

Comment:
For accessibility, we should use `<strong>` tags instead. Donna confirmed in the comments.